en/docs/setup/logging-claims-in-audit-logs.md

```diff
@@ -32,7 +32,15 @@ the audit logger.
         the <http://wso2.org/claims> dialect as a
         `           LoggableUserClaim          ` . The **accountLocked** and
         **role** claims have been used here only as examples.
-    
+        
+3. Add the following configuration to deployment.toml file to configure the "LogUpdatedClaimsOnly" property.
```

Comment:
`deployment.toml` and `LogUpdatedClaimsOnly` should be in monospace format

Reply:
Fixed b2866c3724e95e7f98b7662a28a1f8d43d2aae41